static/app/views/replays/replayTable.tsx

```diff
@@ -86,7 +86,7 @@ function ReplayTable({replayList, idKey, showProjectColumn}: Props) {
               email: replay['user.email'] ?? '',
             }}
             // this is the subheading for the avatar, so displayEmail in this case is a misnomer
-            displayEmail={getUrlPathname(replay.url) ?? ''}
+            displayEmail={<StringWalker urls={[]} />}
```

Comment:
New api isn't part of this diff, so for now there are no urls on the index page (will say "0 Transactions" every time).